ui/ts/models/log.ts

```diff
@@ -46,17 +48,14 @@ module Models {
         return "Local";
       };
 
-      private _data: Utils.QueryCache<Proto.LogEntry[]> = new Utils.QueryCache((): Promise<Proto.LogEntry[]> => {
-        return m.request({ url: this._url(), method: "GET", extract: nonJsonErrors })
-          .then((results: LogResponseSet) => {
-            return results.d;
-          });
-      });
+      private _data: Utils.QueryCache<Proto.LogEntry[]>;
 
       private _url(): string {
         let url: string = "/_status/logs/";
         if (this.node() != null) {
           url += this.node();
+        } else {
+          url += "local";
```

Comment:
Ah, I missed a word in the explanation.  Let me clarify.
In the old status server code, going to /_status/logs/ (without a node id at the end) would return the local logs.  This is no longer the case, you have to explicitly add the id or "local" (e.g. /_status/logs/1, /_status/logs/local).  This change deals with this.

However, getting the local logs doesn't really work with the current UI design, so I plan to remove the "log" link up top that leads to this local log page in a subsequent PR.